hudi-client/hudi-client-common/src/test/java/org/apache/hudi/client/transaction/TestSimpleConcurrentFileWritesConflictResolutionStrategy.java

```diff
@@ -412,6 +412,7 @@ public void testConcurrentWritesWithPendingInstants() throws Exception {
     // make sure c3 has conflict with C1,C11,C12,C4;
     HoodieCommitMetadata currentMetadata = createCommitMetadata(currentWriterInstant, "file-2");
     timeline.reload();
+    metaClient.reloadActiveTimeline();
     List<HoodieInstant> completedInstantsDuringCurrentWriteOperation = TransactionUtils
```

Comment:
What's the difference between `timeline.reload();` and `metaClient.reloadActiveTimeline();`

Reply:
`timeline.reload()` will return a new object, so this line isn't really doing anything since the returned instance is not used. I have removed the `timeline.reload()`